### ISSUE TITLE
Enable Gzip compression for JavaScript files

### ISSUE THREAD

**Issue by TheoChevalier** [mozfr/transvision#578]
We just enabled Gzip server-side, but looks like we can still get some performance by enabling it for JS files as well: http://checkgzipcompression.com/?url=https://transvision.mozfr.org/assets/jquery/jquery.min.js

According to GTmetrix.com:
![capture du 2016-01-04 18-12-30](https://cloud.githubusercontent.com/assets/1294206/12095419/c0f6c2fa-b30e-11e5-9832-b91c0127efd7.png)

**Comment by pascalchevrel** [comment]
I think the whole server would benefit from it, pinging @mozfr/admin-sys

**Comment by MatonAnthony** [comment]
@pascalchevrel Is there a reason to store Jquery ourselves instead of using a CDN ?

**Comment by pascalchevrel** [comment]
yes, hosting it ouselves prevents our users being tracked by the CDN owners (usually Google).

**Comment by MatonAnthony** [comment]
Ok, thanks.
